src/website/src/app/documentation/demos/themes/themes.demo.html

```diff
@@ -275,7 +275,10 @@ <h4 style="margin-bottom: 12px;">Webpack Builds</h4>
             <clr-code-snippet [clrCode]="uiWebpackImports" [clrDisablePrism]="true"></clr-code-snippet>
             <h4 style="margin-bottom: 12px;">Custom Themes</h4>
             <p>If you need to customize components for you application we suggest starting with either the
-                light theme or the dark theme and override variables from there to suite your customizations.</p>
+                light theme or the dark theme and override variables from there to suite your customizations. For
+                now, the best place to find a comprehensive list of component variables is in the
+                <a href="//github.com/vmware/clarity/blob/master/src/clr-angular/utils/_variables.clarity.scss"
+                   target="_blank">_variables.clarity.scss</a> file.</p>
```

Comment:
Not worth worrying about changing but just for note the website has a directive that adds the target blank automatically for external links :)

Reply:
I forgot about that. Will update to use that. It is better.